jwst/datamodels/schemas/filteroffset.schema.yaml

```diff
@@ -27,28 +27,14 @@ allOf:
         are matched with the SCI exposure metadata to get
         values for "row_offset" and "column_offset".
 
-        The option with the "name" key is used for
-        MIRI data.
-      anyOf:
-        - type: array
-          items:
-            type: object
-            properties:
-              name:
-                type: string
-              column_offset:
-                type: number
-              row_offset:
-                type: number
-        - type: array
-          items:
-            type: object
-            properties:
-              filter:
-                type: string
-              pupil:
-                type: string
-              row_offset:
-                type: number
-              column_offset:
-                type: number
+      items:
```

Comment:
I notice that the old schema had an "outer" attribute of:
```
 - type: array
    items:
```
which is not used in the new schema. Is that OK?

Reply:
I think someone more knowledgeable than I about datamodel schemas would have to comment; from comparing this to other schemas I concluded that this outer attribute was probably just a special wrapper required in order to have multiple schema options for both the old miri format file and the new generic file.  With the simplification to just one schema I think it's not necessary, but it would be good to confirm that.

Reply:
Yes, the type needs to be specified at the same level as `description`, as in 

```
filters:
  description: |
    An array of dictionaries where either the
        ...
  type: array
  items:
      type: object
      properties:
         filter:
           type: string
         pupil:
           type: string

```